contents/handbook/growth/marketing/product-announcements.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Product announcements
+sidebar: Handbook
+showTitle: true
+---
+
+Marketing takes responsibility for coordinating and publicising PostHog news, including product announcements. 
```

Comment:
```suggestion
Marketing takes responsibility for coordinating and publicizing news about PostHog, including product announcements. 
```

"PostHog News" seems confusing as it's an internal concept we use to refer to the updates on Monday.